pom.xml

```diff
@@ -58,12 +93,6 @@
 
 
         <!-- TEST DEPENDENCIES -->
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.11</version>
-            <scope>test</scope>
-        </dependency>
```

Comment:
Have a look @ https://plugins.jetbrains.com/plugin/7179-maven-helper , it can help to see in detail the dependencies that are in the project. frequently, dependencies creep in from other dependencies. For example, unit 4 was removed and now it is based on junit 5. However, it is likely that junit 4 is still on the class path, because probably spring is bringing it in, you can exclude it in maven.